site/en/100/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: '#100CoolWebMoments'
+description: 'Take a stroll down memory lane and celebrate #100CoolWebMoments since Chrome’s first release.'
+thumbnail: image/x1Los57vDga6OEMNi1dIJwZ0qvp2/1gyOFYMC1sFOBkC5csD3.png
+alt: 'Chrome 100'
+layout: 'layouts/chrome-100.njk'
+type: landing
```

Comment:
Can you please [add `draft: true` to the YAML frontmatter](https://github.com/GoogleChrome/developer.chrome.com/blob/295ac24659cb8cba11660486cf706096a50d37b8/site/site.11tydata.js#L7-L12)? We want to be able to merge this PR without `/100/` going live until we're ready.

Reply:
When we added this in our index.md file, our Preview link stopped working so did not add it right now.